.gitignore

```diff
@@ -10,6 +10,7 @@ config.yml
 /web/xhprof
 /web/widgets/layout
 .DS_STORE
+/.vscode
```

Comment:
Stuff like `/.vscode` should actually go in your own [global gitignore](https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore). Otherwise we end up having to add a lot of IDE config directories like `.idea` etc as well. Same goes for the `.DS_STORE` that's above it. Normally you only list files that are relevant to the project but should not be tracked in git to the project's `.gitignore`. For example, config files. For tooling and OS files, you ignore those in your own `.gitignore` because those can differ a lot from developer to developer. And that way you also don't have to ignore them for every project you work on.

Reply:
Ha so even `.idea` is actually ignored explicitely in this project. So I definitely wouldn't blame you for not knowing this. Feel free to remove that line as well.

Reply:
Quickly resolved it so this can be merged